filebeat/input/filestream/filestream.go

```diff
@@ -256,6 +265,6 @@ func (f *logFile) handleEOF() error {
 func (f *logFile) Close() error {
 	f.readerCtx.Cancel()
 	err := f.file.Close()
-	f.tg.Stop() // Wait until all resources are released for sure.
+	_ = f.tg.Stop() // Wait until all resources are released for sure.
```

Comment:
[Question]

Any reason not to log this error?

Reply:
We cannot really act on it but if you think it's somehow important I can add it.